src/Evolution/Systems/NewtonianEuler/SoundSpeedSquared.hpp

```diff
@@ -0,0 +1,40 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/NewtonianEuler/Tags.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+
+// IWYU pragma: no_forward_declare hydro::Tags::EquationOfState
+
+namespace NewtonianEuler {
+/// Computes the Newtonian sound speed squared
+/// \f$c_s^2 = \chi + p\kappa / \rho^2\f$, where
+/// \f$p\f$ is the fluid pressure, \f$\rho\f$ is the mass density,
+/// \f$\chi = (\partial p/\partial\rho)_\epsilon\f$ and
+/// \f$\kappa = (\partial p/ \partial \epsilon)_\rho\f$, where
+/// \f$\epsilon\f$ is the specific internal energy.
+template <typename DataType, typename EquationOfStateType>
+Scalar<DataType> sound_speed_squared(
+    const Scalar<DataType>& mass_density,
+    const Scalar<DataType>& specific_internal_energy,
+    const EquationOfStateType& equation_of_state) noexcept;
+
+namespace Tags {
+/// Compute item for the sound speed squared \f$c_s^2\f$.
```

Comment:
(optional) It might be good to have the docs point to the function with the `\see NewtonianEuler::sound_speed_squared`